### PR TITLE
Created preprocessing script for French version of legislation from Luxembourg and preprocessed legislation

### LINKED ISSUE #79: Develop preprocessing script for legislation from Luxembourg

### PR DESCRIPTION
**Pull request by danpdraper**
The legislation in this pull request is the [*Law Concerning the Protection of Nature and Natural Resources* of 1982](https://legilux.public.lu/eli/etat/leg/loi/1982/08/11/n1/jo). I decided to use the 1982 Law instead of the *Law on Modification of the Law of July 29 1965 Concerning Conservation of Nature and Natural Resources* of 1978 for three reasons:

* the 1982 Law was enacted only four years after the 1978 Law;
* the 1982 Law repeals the 1978 Law and the 1965 Law;
* the 1982 Law is a full restatement of the nature protection code, whereas the 1978 Law only contains modifications of the 1965 Law.

Closes #79

### CODE REVIEW

File: raw_data/preprocessed/europe/luxembourg_french.txt [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+(4) Dans la zone verte, sont soumis à l'autorisation du Ministre l'ouverture de minières, sablières, carrières ou gravières ainsi que l'enlèvement de terre végétale sur une superficie dépassant un are, et le dépôt de déblais d'un volume dépassant cinquante m3. Sauf dispense du Ministre, le bénéficiaire de l'autorisation est tenu de rendre au sol son caractère naturel, en boisant ou en regarnissant de végétation les excavations, déblais ou remblais destinés à subsister d'une manière permanente. Les plantations sont exécutées dans la mesure où l'avancement des travaux d'exploitation le permet. Le Ministre constate, sur le rapport de l'Administration des Eaux et Forêts, la possibilité de reboiser ou de regarnir et impartit au maître d'oeuvre un délai endéans lequel les travaux doivent être exécutés et terminés. Faute par l'intéressé de se conformer à l'injonction du Ministre, celui-ci charge l'Administration de l'exécution des travaux aux frais du contrevenant. Le recouvrement des frais se fait comme en matière domaniale.
```
Comment (Skyrolon) [inline raw_data/preprocessed/europe/luxembourg_french.txt:11]: Should the instance of "d'oeuvre" be "d'œuvre"?

File: raw_data/preprocessed/europe/luxembourg_french.txt [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+(13) Il est interdit de planter des résineux à une distance inférieure à quatre mètres du bord des cours d'eau.
+
+(14) Il est interdit de réduire, de détruire ou de changer les biotopes tels que mares, marécages, marais, couvertures végétales constituées par des roseaux ou des joncs, haies, broussailles ou bosquets. Sont interdits pendant la période du premier mars au 30 septembre: [a] la taille des haies vives et des broussailles à l'exception de la taille des haies servant à l'agrément des maisons d'habitation ou des parcs, ainsi que de celle rendue nécessaire par des travaux effectués dans les peuplements forestiers; [b] l'essartement à feu courant et l'incinération de la couverture végétale des prairies, friches ou bords de champs, de prés, de terrains forestiers, de chemins et de routes. Le Ministre peut exceptionnellement déroger à ces interdictions pour des motifs d'intérêt général. Un règlement grand-ducal arrêtera les modalités concernant l'incinération des pailles et des chaumes.
```
Comment (Skyrolon) [inline raw_data/preprocessed/europe/luxembourg_french.txt:33]: Microsoft word suggests "prés" should be "près".

File: raw_data/preprocessed/europe/luxembourg_french.txt [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+(32) L'Administration des Eaux et Forêts veillera à la réalisation et au respect des plans de gestion.
+
+CHAPITRE 5 - Subventions en faveur de la protection de l'espace rural et des forêts
+
+(33) Un règlement grand-ducal déterminera les conditions d'octroi et les montants de subventions pour travaux dans l'intérêt de la conservation du caractère et de la beauté de l'espace rural et des forêts. Peuvent être subventionnés: le maintien ou la restauration de prés de vallées à l'intérieur de massifs forestiers; la protection des végétations dans les sites rocheux et les escarpements; la protection des végétations bordant les cours d'eau et des zones tourbeuses; la plantation de haies et de bosquets; la protection de la forêt et l'amélioration des structures forestières; les aménagements et les ouvrages prévus à l'article 28; les modifications des ouvrages résultant des dispositions de l'article 7.
```
Comment (Skyrolon) [inline raw_data/preprocessed/europe/luxembourg_french.txt:75]: Microsoft word suggests "prés" should be "près".

File: raw_data/preprocessed/europe/luxembourg_french.txt (hunk 1 above)
Comment (danpdraper) [reply raw_data/preprocessed/europe/luxembourg_french.txt:11]: No, under issue 90 I am going to convert all instances of œuvre across the entire body of legislation to oeuvre.

File: raw_data/preprocessed/europe/luxembourg_french.txt (hunk 2 above)
Comment (danpdraper) [reply raw_data/preprocessed/europe/luxembourg_french.txt:33]: In both this case and the case below, `prés` is actually correct. `près` means "near" while `prés` is the plural form of `pré`, which means "meadow". If you throw either of these articles into Google Translate, you will see that "meadows" more appropriately fits the context.

**Pull request merged**